gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java

```diff
@@ -141,96 +155,117 @@ public Builder toBuilder() {
     /**
      * The TotalTimeout parameter has ultimate control over how long the logic should keep trying
      * the remote call until it gives up completely. The higher the total timeout, the more retries
-     * can be attempted.
+     * can be attempted. The default value is {@code Duration.ZERO}.
      */
     public abstract Builder setTotalTimeout(Duration totalTimeout);
 
     /**
      * The InitialRetryDelay parameter controls the delay before the first retry. Subsequent retries
-     * will use this value adjusted according to the RetryDelayMultiplier.
+     * will use this value adjusted according to the RetryDelayMultiplier. The default value is
+     * {@code Duration.ZERO}.
      */
     public abstract Builder setInitialRetryDelay(Duration initialDelay);
 
     /**
      * The RetryDelayMultiplier controls the change in retry delay. The retry delay of the previous
      * call is multiplied by the RetryDelayMultiplier to calculate the retry delay for the next
-     * call.
+     * call. The default value is {@code 1.0}.
      */
     public abstract Builder setRetryDelayMultiplier(double multiplier);
 
     /**
      * The MaxRetryDelay puts a limit on the value of the retry delay, so that the
-     * RetryDelayMultiplier can't increase the retry delay higher than this amount.
+     * RetryDelayMultiplier can't increase the retry delay higher than this amount. The default
+     * value is {@code Duration.ZERO}.
      */
     public abstract Builder setMaxRetryDelay(Duration maxDelay);
 
     /**
-     * MaxAttempts defines the maximum number of attempts to perform. If number of attempts reaches
-     * this limit the logic will give up retrying even if the total retry time is still lower than
-     * TotalTimeout.
+     * The MaxAttempts defines the maximum number of attempts to perform. If number of attempts
+     * reaches this limit the logic will give up retrying even if the total retry time is still
+     * lower than TotalTimeout. The default value is {@code 0}.
      */
     public abstract Builder setMaxAttempts(int maxAttempts);
 
     /**
+     * The Randomized defines if the delay time should be randomized using the provided delay value
+     * as an upper bound of the randomized value, which in its turn will be used as an actually
+     * applied delay time. The default value is {@code true}.
+     */
+    public abstract Builder setRandomized(boolean randomized);
```

Comment:
I'm not sure what you mean by this, but from wikipedia: "Jitter is a significant, and usually undesired, factor in the design of almost all communications links". I especially don't like the "undesired".